packages/web-twig/src/Resources/components/TextArea/README.md

```diff
@@ -34,7 +34,7 @@ Without lexer:
     id: "example",
     isRequired: true,
     label: "Label",
-    maxLength: 180,
+    maxlength: 180,
```

Comment:
This syntax is currently valid in Twig, but what about consistency with React where is used `maxLength`? We can unify naming as well, or can't we?

Reply:
This is only a problem of the type, I think.

Reply:
Please see https://github.com/lmc-eu/spirit-design-system/pull/896 where I solved this also for the web-react package.

Reply:
Lovely, thank you. ❤️